iModelCore/ECDb/ECDb/ECSql/Parser/SqlBison.cpp

```diff
@@ -2442,6 +2442,7 @@ YYSTYPE yylval YY_INITIAL_VALUE (= yyval_default);
 
     /* Number of syntax errors so far.  */
     int yynerrs = 0;
+    UNUSED_VARIABLE(yynerrs);

```

Comment:
This is generated file. I logged an issue for that.

https://github.com/iTwin/imodel-native-internal/issues/638

Reply:
Are you ok with this modification for now?

Reply:
yes, it's fine for now.